CONTRIBUTING.md

```diff
@@ -31,7 +31,7 @@ See [Developer Handbook][developer-handbook] for more information about developm
 
 ## Commit Conventions
 
-All commits you make SHOULD adhere to our commit guidelines. We use [conventional commits][conventional-commits] strategy with a slight modification of our own - [@lmc-eu/commitlint-config][commitlint-config]. This is later used during the release phase to determine how to bump the packages' version numbers based on commit history. 🚀
+All commits you make SHOULD adhere to our commit guidelines. We use [conventional commits][conventional-commits] strategy with a slight modification of our own - [@lmc-eu/commitlint-config][commitlint-config]. This is later used during the release phase to determine how to bump the packages’ version numbers based on commit history. 🚀
```

Comment:
The correct apostrophe sign is important for a great typography, and I have always been rooting for a great typography.

But this is a technical document and I'm afraid people (I mean us) will struggle with the correct styling. If we are going to check the correct apostrophe character, I'd also check quotes and dashes, to name a few. But we cannot force people to use them, IMO.